packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -11,12 +11,15 @@ const CrossmintContext = createContext<CrossmintContext | null>(null);
 
 export function CrossmintProvider({
     children,
-    ...createCrossmintParams
-}: { children: ReactNode } & Parameters<typeof createCrossmint>[0]) {
+    apiKey,
+    overrideBaseUrl,
+}: Omit<Crossmint, "jwt"> & {
+    children: ReactNode;
+}) {
     const [version, setVersion] = useState(0);
 
     const crossmintRef = useRef<Crossmint>(
-        new Proxy<Crossmint>(createCrossmint(createCrossmintParams), {
+        new Proxy<Crossmint>(createCrossmint({ apiKey, overrideBaseUrl }), {
```

Comment:
it makes sense to keep `jwt` in the Crossmint type as the provider return type is Crossmint and returns a jwt, but shouldn't we also remove `jwt` from the `createCrossmint` helper fn?

Reply:
or maybe if other devs are using the function from common-sdk-base then maybe we should keep it 🤔